roles/de-image-janitor/tasks/config

```diff
@@ -0,0 +1,11 @@
+---
+config:
+  DE:
+    AMQP:
+      User: "ego"
+      Password: "secret"
+      Host: "host"
+      Port: 1234
+      Vhost: "none"
+  BaseULRs:
+    Apps: "app"
```

Comment:
is this hardcoded? if yes we should use template also to be able to change these values.